### DISCUSSION TITLE
Set default for code blocks to be `.column-screen-inset-right`?

### DISCUSSION THREAD

**Discussion by jhchou** [quarto-dev/quarto-cli#3313] [Q&A] [answered]
I would like code blocks in Quarto documents to **default** to a wider layout than `.column-body`. I've been getting the effect I'm trying to achieve with:

    ```{julia}
    #| column: screen-inset-right
    findfirst(==(-1), cumsum([c == '(' ? 1 : -1 for c in input])) # find index of first matching value
    ```

But with many code blocks, it gets a bit repetitive to include `#| column: screen-inset-right` in every code chunk. Is there a nice way to set code chunks to have a different default layout?

(If not possible to set a default, I also tried to cut down the use of vertical space with the old RMarkdown style of `{julia column=screen-inset-right}`, but not surprisingly, it's not supported.)

**Comment by dragonstyle** [comment, marked as answer]
Right now there isn't a way to provide a global, default column for code blocks (though this is a great feature for Quarto to support).

**Reply by jhchou** [reply]
Thanks for the response!

I just wanted to ask the question because I've so frequently missed a well-documented supported solution that it was just as likely that it _was_ already possible. In any case, the per-chunk solution is still just fine.

**Reply by dragonstyle** [reply]
Definitely - we've had at least one other request for this (and I took a quick look thinking it would be simple, but it was a little more complicated to support than I initially thought). I believe we have an open issue to support this, but I'm having trouble finding it right now, otherwise I'd link to it!
